apps/challenges/urls.py

```diff
@@ -166,4 +166,14 @@
         views.validate_challenge_config,
         name="validate_challenge_config",
     ),
+    url(
+        r"^(?P<challenge_pk>[0-9]+)/get_worker_logs/$",
+        views.get_worker_logs,
+        name="get_worker_logs"
+    ),
+    url(
+        r"^(?P<challenge_pk>[0-9]+)/manage_worker/(?P<action>[\w-]+)$",
```

Comment:
`r"^(?P<challenge_pk>[0-9]+)/manage_worker/(?P<action>[\w-]+)$",` -> `r"^(?P<challenge_pk>[0-9]+)/manage_worker/(?P<action>[\w-]+)/$"`,

Reply:
Forward slash missing at the end of URL.